e2e-tests/portal/cypress/fixtures/mockUtilisationReportDetails.js

```diff
@@ -84,10 +84,17 @@ const generateUpToDateReportDetails = () => {
 
 const upToDateReportDetails = generateUpToDateReportDetails();
 
+const tfmFacilityForReport = {
+  facilitySnapshot: {
+    ukefFacilityId: '20001371', // this is taken from the fixtures 'valid-utilisation-report-<...>.xlsx' files
```

Comment:
I disagree, this is the first time we are explicitly using a facility id from a test utilisation report xlsx file and I think it could save a lot of time debugging in the future/keeping these tests maintainable if devs know where this comes from. I think I will need to add a lot more facilities however based on the other comment re the end to end tests for the messages to be a decent test case so I will reword the comment due to that anyway.

Reply:
Can we at least mention the specific XLSX file name please - appreciate this could be helpful for future debugging - currently if I search for the UKEF facility ID in the codebase, I cannot see any results in an XLSX file.